<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qMRMLPlotSeriesPropertiesWidget</class>
 <widget class="qMRMLWidget" name="qMRMLPlotSeriesPropertiesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>169</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="1" column="0">
    <widget class="QLabel" name="InputTableLabel">
     <property name="text">
      <string>Input Table:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="qMRMLNodeComboBox" name="inputTableComboBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="nodeTypes">
      <stringlist notr="true">
       <string>vtkMRMLTableNode</string>
      </stringlist>
     </property>
     <property name="showHidden">
      <bool>true</bool>
     </property>
     <property name="noneEnabled">
      <bool>false</bool>
     </property>
     <property name="addEnabled">
      <bool>true</bool>
     </property>
     <property name="removeEnabled">
      <bool>true</bool>
     </property>
     <property name="editEnabled">
      <bool>true</bool>
     </property>
     <property name="renameEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="plotTypeLabel">
     <property name="text">
      <string>Plot Type:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="plotTypeComboBox">
     <property name="toolTip">
      <string/>
     </property>
     <item>
      <property name="text">
       <string>line</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>bar</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>scatter</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>scatter bar</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="xAxisLabel">
     <property name="text">
      <string>X Axis Column:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="xAxisComboBox">
     <property name="toolTip">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="yAxisLabel">
     <property name="text">
      <string>Y Axis Column:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="yAxisComboBox">
     <property name="toolTip">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="labelsComboBox">
     <property name="toolTip">
      <string>This column contains a label for each data point. Label is displayed in the tooltip when the mouse hovers over a data point in the plot view.</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QComboBox" name="markersStyleComboBox">
     <property name="toolTip">
      <string/>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>none</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>cross</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>plus</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>square</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>circle</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>diamond</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="ctkDoubleSpinBox" name="markersSizeDoubleSpinBox">
     <property name="value">
      <double>10.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QComboBox" name="lineStyleComboBox">
     <property name="toolTip">
      <string/>
     </property>
     <item>
      <property name="text">
       <string>none</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>solid</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>dash</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>dot</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>dash-dot</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>dash-dot-dot</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="ctkDoubleSpinBox" name="lineWidthDoubleSpinBox">
     <property name="value">
      <double>4.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="ctkColorPickerButton" name="plotSeriesColorPickerButton">
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="color">
      <color>
       <red>85</red>
       <green>170</green>
       <blue>255</blue>
      </color>
     </property>
     <property name="displayColorName">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="labelsLabel">
     <property name="text">
      <string>Labels Column:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="markerStyleLabel">
     <property name="text">
      <string>Markers Style:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="markersSizeLabel">
     <property name="text">
      <string>Markers Size:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="lineStyleLabel">
     <property name="text">
      <string>Line Style:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="lineWidthLabel">
     <property name="text">
      <string>Line Width:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="plotSeriesColorLabel">
     <property name="text">
      <string>Color:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkColorPickerButton</class>
   <extends>QPushButton</extends>
   <header>ctkColorPickerButton.h</header>
  </customwidget>
  <customwidget>
   <class>ctkDoubleSpinBox</class>
   <extends>QWidget</extends>
   <header>ctkDoubleSpinBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>qMRMLPlotSeriesPropertiesWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>inputTableComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>356</y>
    </hint>
    <hint type="destinationlabel">
     <x>228</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
